### PR TITLE
Added Pattern Lab and SASS skeleton and build scripts

### PR DESCRIPTION
**Pull request by james-nash**
Copied the gulpfile from Matteo's PL starter project and slightly modified some of the file paths based on the comments in https://digitalrig.atlassian.net/wiki/display/GRAV/UI+lib+considerations

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Exlclude build Output
+dist/
+
+
+######### App / OS-specific guff ##########
+
 ### macOS ###
 *.DS_Store
```
Comment (james-nash) [inline .gitignore]: Good spot.

This was auto-generated by https://www.gitignore.io/ . Weirdly, they don't have the `*` on other macOS ignores, so it looks deliberate. However, I can't find any information suggesting that the file is ever called anything other than just `.DS_Store`.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+ * The gulp wrapper around patternlab-node core, providing tasks to interact with the core library and move supporting frontend assets.
+******************************************************/
+const gulp = require('gulp');
+const path = require('path');
+const browserSync = require('browser-sync').create();
+const sass = require('gulp-sass');
+const eyeglass = require('eyeglass');
+const argv = require('minimist')(process.argv.slice(2));
+const chalk = require('chalk');
+const concat = require('gulp-concat');
```
Comment (james-nash) [inline gulpfile.js]: Are we not?
Sure, the location and name of `pattern-scaffolding.css` may change, but I'd have thought we will have one.

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,306 @@
+/******************************************************
+ * PATTERN LAB NODE
```
Comment (james-nash) [inline gulpfile.js]: Yup. On it.

File: gulpfile.js (hunk 2 above)
Comment (james-nash) [inline gulpfile.js]: Ah. Didn't quite understand this at first, but now I do. On it.

**Comment by ThePeach** [comment]
👍

**Pull request merged**